gw_eccentricity/eccDefinition.py

```diff
@@ -1435,7 +1544,7 @@ def derivative_of_eccentricity(self, t, n=1):
             nth order time derivative of eccentricity.
         """
         # Check that t is within tmin and tmax to avoid extrapolation
-        self.check_time_limits(t)
+        self.check_input_limits(t, self.tmin, self.tmax)
```

Comment:
Do these changes have anything to do with the current PR?

Reply:
One of the comments earlier was to not copy lines and make use of functions. I found that I was repeating the checks of time and frequency inputs in the new function where we set ecc and ano to zero. So I just modified this function so that this can be now used for both time and frequency inputs.

Reply:
Ok, fine for now, but that's a perfect example of: make an issue and deal with later.

Reply:
Ok, will do that for future PRs.